_commands/basics/rm.md

```diff
@@ -0,0 +1,55 @@
+---
+---
+
+rm
+--
+
+`rm` is used to delete one or multiple files from your current directory. 
+
+~~~ bash
+$ rm file1.txt file2.txt
+
```

Comment:
There's trailing whitespace here that I'm personally not a fan of – remove this extra line for all of your code blocks? 

<img width="1021" alt="screen shot 2016-03-24 at 2 46 37 pm" src="https://cloud.githubusercontent.com/assets/5882053/14027678/4293b364-f1cf-11e5-88eb-1e09a070e91e.png">